### PR TITLE
UI: Changed UI of reactions-summary dialog box

### PR DESCRIPTION
**Pull request by Ahmedgagan**
@jjaffeux

### CODE REVIEW

File: assets/stylesheets/common/discourse-reactions.scss [hunk 1]
```diff
@@ -130,60 +130,103 @@ html.discourse-reactions-no-select {
...
+    justify-content: space-between;
+  }
+
+  .count {
+    color: var(--primary);
+    font-size: $font-up-1;
+    margin-left: 0.5em;
+  }
+
+  .hide-users {
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-reactions.scss:174]: why all the !important here ?

File: assets/stylesheets/common/discourse-reactions.scss [hunk 2]
```diff
@@ -130,60 +130,103 @@ html.discourse-reactions-no-select {
...
-      color: var(--primary-high);
-    }
+  .reaction-wrapper {
+    display: flex;
+    grid-column: 1 / 4;
   }
 
-  &:not(.is-displayed) {
-    filter: grayscale(1);
+  .show-users {
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-reactions.scss:214]: why all the !important here ?

File: assets/stylesheets/common/discourse-reactions.scss (hunk 1 above)
Comment (Ahmedgagan) [reply assets/stylesheets/common/discourse-reactions.scss:174]: Because It was using its specific CSS colors and was not accessing our CSS

Comment (jjaffeux) [reply assets/stylesheets/common/discourse-reactions.scss:174]: Yes I know what's the effect of important is :) I will have look after merging and see if I can't change this, maybe specificity in core is too high.

**Pull request merged**